### PR TITLE
Update: Make 404 pages support multiple languages, and increase the length of the article summary

### PR DESCRIPTION
**Pull request by FrederickBun**
Previously, 404 pages were only in Chinese, but now translation has been added to make them support all languages of the topic
Before, the article summary was very small, resulting in very little information, now increased to 300, the content will be more rich

## Summary by CodeRabbit

- **New Features**
	- Added "Go Back Home" navigation option in English, Japanese, Simplified Chinese, and Traditional Chinese for improved localization.
	- Updated the 404 error page with clearer navigation options, including a link to return to the previous page and a new option to go back to the homepage.
	- Increased content preview length in post display for better context before reading full articles.

- **Bug Fixes**
	- Improved clarity and functionality of navigation links on the 404 error page.

### CODE REVIEW

**Comment by FrederickBun** [comment]
@coderabbitai review

File: layout/_partials/404-template.ejs
Comment (EvanNotFound) [inline layout/_partials/404-template.ejs:1]: The code will not run correctly as the git conflict is not solved

**Review by EvanNotFound — COMMENTED** [review]
Please fix the code here

**Review by EvanNotFound — CHANGES_REQUESTED** [review]
Should use `__("go-back-home")` helper function in order to correctly display i18n title

**Comment by FrederickBun** [comment]
> Should use helper function in order to correctly display i18n title`__("go-back-home")`

OK

**Comment by FrederickBun** [comment]
> Should use helper function in order to correctly display i18n title`__("go-back-home")`

an how can i do

**Pull request merged**